WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Quick Start/QuickStartChecklistView.swift

```diff
@@ -0,0 +1,159 @@
+
+    var tours: [QuickStartTour] = []
+    var blog: Blog?
+    var onTap: (() -> Void)?
+
+    private lazy var mainStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [
+            labelStackView,
+            progressIndicatorView
+        ])
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        stackView.distribution = .fill
+        stackView.spacing = Metrics.mainStackViewSpacing
+        return stackView
+    }()
+
+    private lazy var labelStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [
+            titleLabel,
+            subtitleLabel
+        ])
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.spacing = Metrics.labelStackViewSpacing
+        return stackView
+    }()
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = WPStyleGuide.serifFontForTextStyle(.body, fontWeight: .semibold)
+        label.textColor = .text
+        return label
+    }()
+
+    private lazy var subtitleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = WPStyleGuide.fontForTextStyle(.callout)
+        label.textColor = .textSubtle
+        return label
+    }()
+
+    private lazy var progressIndicatorView: ProgressIndicatorView = {
+        let appearance = ProgressIndicatorView.Appearance(
+            size: Metrics.progressIndicatorViewSize,
+            lineColor: .primary,
+            trackColor: .separator
+        )
+        let view = ProgressIndicatorView(appearance: appearance)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.setContentHuggingPriority(.defaultHigh, for: .horizontal)
+        view.isAccessibilityElement = false
+        return view
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+        startObservingQuickStart()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("Not implemented")
+    }
+
+    deinit {
+        stopObservingQuickStart()
+    }
+
+    func configure(tours: [QuickStartTour], blog: Blog, title: String, hint: String) {
+        self.tours = tours
+        self.blog = blog
+        titleLabel.text = title
+        accessibilityHint = hint
+        updateViews()
+    }
+}
+
+extension QuickStartChecklistView {
+
+    private func setupViews() {
+        addSubview(mainStackView)
+        pinSubviewToAllEdges(mainStackView, insets: Metrics.mainStackViewInsets)
+
+        let tap = UITapGestureRecognizer(target: self, action: #selector(didTap))
+        addGestureRecognizer(tap)
+
+        isAccessibilityElement = true
+        accessibilityTraits = .button
```

Comment:
A tiny suggestion would be to group all accessibility logic in one helper and call it from `configure`. Currently, it's split over three places. Feel free to ignore this though 👍

Reply:
Fixed! a2e4c5f

I still need to set the accessibilityLabel in `updateViews()` -- otherwise, the "x out of y completed" doesn't get updated.